### PR TITLE
Update net framework to 4.6.1 and nuget dependencies #151

### PR DESCRIPTION
**Pull request by seanfarmar**

### CODE REVIEW

File: src/HashBus.Viewer/packages.config [hunk 1]
```diff
@@ -4,5 +4,5 @@
   <package id="GitVersionTask" version="3.6.2" targetFramework="net451" developmentDependency="true" />
   <package id="LiteGuard" version="0.10.0" targetFramework="net451" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net451" />
```
Comment (adamralph) [inline src/HashBus.Viewer/packages.config]: Should these other entries also be changed to `net461`?

Comment (seanfarmar) [reply src/HashBus.Viewer/packages.config]: yeah... I'll recheck this

**Comment by seanfarmar** [comment]
@adamralph - updated

**Comment by seanfarmar** [comment]
@adamralph squashed to one commit

**Pull request merged**